src/main/kotlin/StructuredDataGenerators/JsonGenerator.kt

```diff
@@ -0,0 +1,14 @@
+package StructuredDataGenerators
+
+import org.json.JSONObject
+
+class JsonGenerator {
+    private val jsonObj = JSONObject()
+
+    fun generateJson(mapOfKeysToValues: Map<String, Any>): String {
```

Comment:
nitpick: looks like your output is technically a JSON string, not a JSON.

Reply:
Thanks for pointing that out! It would make more sense to change the method name from `generateJson` to generateJsonString`